raiden/tests/integration/rpc/assumptions/test_rpc_gas_price_assumptions.py

```diff
@@ -28,30 +28,97 @@ def increasing_gas_price_strategy(web3: Web3, _transaction_params: Dict) -> GasP
     return increasing_gas_price_strategy
 
 
-def test_duplicated_transaction_same_gas_price_raises(deploy_client: JSONRPCClient) -> None:
-    """ If the same transaction is sent twice a JSON RPC error is raised. """
-    gas_price = GasPrice(2000000000)
-    gas_price_strategy = make_fixed_gas_price_strategy(gas_price)
-    deploy_client.web3.eth.setGasPriceStrategy(gas_price_strategy)
+def test_resending_pending_transaction_raises(deploy_client: JSONRPCClient) -> None:
```

Comment:
The changes to this test are cosmetic, so that it matches the style of the other tests.